ios/MobiusMobileWallet.xcodeproj/project.pbxproj

```diff
@@ -25,6 +24,20 @@
 		C2C418DC3CD144C6875373C8 /* Roboto.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 89F9404328D54B39B885FE8B /* Roboto.ttf */; };
 		D042CDFFE93C4FFE9410B3C6 /* Helvetica Neue.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 97DB4B87A3CE4721975E2AB5 /* Helvetica Neue.ttf */; };
 		F6D0D10901774F07A8FBA2E3 /* Helvetica Neue Italic.ttf in Resources */ = {isa = PBXBuildFile; fileRef = D1A4434F98F742D9B1FA9409 /* Helvetica Neue Italic.ttf */; };
+		73CF083B00164B66AF787D34 /* Entypo.ttf in Resources */ = {isa = PBXBuildFile; fileRef = FE7055765602416EA54B58F9 /* Entypo.ttf */; };
```

Comment:
These lines are autogenerated but we don't need all of this fonts in the release bundle, so let's revert it.